### DISCUSSION TITLE
Vercel ip is blocked from my ISP

### DISCUSSION THREAD

**Discussion by omarwahbi** [vercel/community#5202] [Help]
### Summary

Hi,
My website is deployed and has its own domain and was working just fine but now ISP across all the country blocked 76.76.21.21 ip address and now the website is inaccessible is there anyway way around it? Can i get a different ip address? 
ps: BTW i can open the website by running vpn!
Thank you in advance.

### Steps to Reproduce

Visit 
https://www.silsilat-alibda.com/
and it should be working just fine for you but in my country i get this (screenshot attached)
![IMG_5212](https://github.com/vercel/community/assets/99210606/d20105ed-6bfd-4749-abfb-8c3997f821d2)

**Comment by michael-mb** [comment]
I have the same issue but in germany.

**Reply by omarwahbi** [reply]
I just switched to google firebase hosting 🤦🏻‍♂️
